trunk-recorder/main.cc

```diff
@@ -1197,7 +1221,20 @@ void monitor_messages() {
   while (1) {
 
     if (exit_flag) { // my action when signal set it 1
-      printf("\n Signal caught!\n");
+      BOOST_LOG_TRIVIAL(info) << "Signal caught! Closing calls...";
```

Comment:
Nice - this is a good idea. It looks like this should clean up things and shutdown all the open calls.

Reply:
Yeah, I hated ending up with orphan WAV files (unconverted and with no corresponding JSON file).

Reply:
This is a great feature - lets keep it in the PR